ta/pkcs11/src/object.c

```diff
@@ -541,18 +568,31 @@ enum pkcs11_rc entry_find_objects_init(struct pkcs11_client *client,
 	 * candidates that match caller attributes.
 	 */
 
-	LIST_FOREACH(obj, &session->object_list, link) {
-		if (check_access_attrs_against_token(session, obj->attributes))
-			continue;
-
-		if (!attributes_match_reference(obj->attributes, req_attrs))
-			continue;
-
-		rc = find_ctx_add(find_ctx, pkcs11_object2handle(obj, session));
-		if (rc)
-			goto out;
+	/* scan all session objects first */
```

Comment:
s/scan/Scan/

ditto for the other inline comment: start sentences with an uppercase letter.

Reply:
OK